workspaces/desktop/server/package.json

```diff
@@ -3,16 +3,13 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@novnc/novnc": "^1.4.0",
-    "command-line-args": "^5.2.1",
-    "command-line-usage": "^7.0.1",
-    "express": "^4.18.2",
+    "@novnc/novnc": "^1.5.0",
```

Comment:
I have to back out this bump; 1.5.0 stopped publishing the ESM code that we relied on being able to import.

https://github.com/novnc/noVNC/issues/1792